.github/workflows/ci.yml

```diff
@@ -31,6 +31,8 @@ jobs:
             cmake_opts: -DBUILD_WARNINGS_AS_ERRORS=On -DUSE_BUNDLED_DEPS=False -DMINIMAL_BUILD=True
           - name: sanitizers
             cmake_opts: -DUSE_ASAN=On -DUSE_UBSAN=On -DUSE_BUNDLED_DEPS=False
+          - name: zig
+            cmake_opts: -DUSE_BUNDLED_DEPS=True -DUSE_BUNDLED_LIBELF=Off -DCREATE_TEST_TARGETS=Off
```

Comment:
Build with bundled deps enabled and with system libelf to mimic what Falco will do.
Also, do not build test, just sinsp-example since:
* zig build will only be made for release artifacts of Falco
* Falco will run its e2e test framework on its zig builds (https://github.com/falcosecurity/falco/pull/3307)
* There is a build failure because our tests still use ` __gnu_cxx::stdio_filebuf` somewhere (see https://github.com/falcosecurity/libs/pull/2037)